### DISCUSSION TITLE
2023 Make Marketing Team Goals-- Open Discussion

### DISCUSSION THREAD

**Discussion by santanainniss** [WordPress/Marketing-Team#165] [General]
Teams around the community often post yearly goals to guide work and impact over the year, and then year end posts recapping all the accomplishments of the year!

We're opening a call for open discussion to help set the goals and vision for the team over the next calendar year. This discussion will stay open until **January 30, 2023.** We will then discuss during the **January 31, **2023**** Marketing team meeting. Team Reps will compile and post the team goals to our team blog no later than February 15, 2023. 

Some guiding questions:

- What would you like to accomplish as a team in 2023?
- What are some important issues facing the team?
- What are some things you wish the team did/could do? 
- What types of collaboration would be good in 2023?

Don't be shy! ALLLL thoughts and ideas are welcome and appreciated.

**Comment by santanainniss** [comment]
Kicking off the conversation here, one thing that I am very exited to collaborate on is on simplifying processes like note taking! I am also really excited to collaborate on finding impactful and straightforward good first issues for new contributors. I love the idea of breaking down barriers to contribution for first timers. Our team reps from 2022 (and before!) have done such great work getting the team to where it is today. So I am excited to see how we can build on those tremendous contributions to drive impact for the Project and our team!

**Comment by jpantani** [comment]
Hi @santanainniss! I think the [Grow Your Story campaign](https://github.com/WordPress/Marketing-Team/issues/51) has reached a critical mass of user-generated contributions. I've found in my marketing experience that some of the best marketing comes from people who love your product(s). I would love to find a community collaborator and even a community squad who can help bring to life some of those stories. I think these stories will really highlight the value of WP for folks who are new to WordPress are folks who have been around WordPress for a while and have made such an impact in the software/community or, conversely, how the software/community has impacted them! It seems like @murillol expressed some interest here so pls advise how this can fit into the goals. Seemed like it was also a popular activity for WordCamp contributor days as people could furnish us with their own first-person narratives.

**Reply by nomad-skateboarding-dev** [reply]
I really like this idea!

Is there already a connection with [Topher at HeroPress](https://heropress.com/)?

If not, I feel coordinating with HeroPress would give years of stories to pull from, as well another space for those new to WordPress to begin to engage with the community.

I can volunteer in any way with that connection, coordination, or implementation of anything that comes from it. 🙂

**Reply by jpantani** [reply]
Have definitely worked with Topher at the onset of the campaign to convey the two would have slightly different approaches. GYS is focused more on getting social media-sized sound bites while Hero Press focused on a more well-rounded viewpoint. I think this campaign is getting ready for new life though. [Follow the fun on this Grow Your Story GitHub ticket](https://github.com/WordPress/Marketing-Team/issues/51)

**Reply by abhansnuk** [reply]
Hello @nomad-skateboarding-dev good question. There are quite a few areas connecting with Topher from my understanding. One of them in People of WordPress, which provides some or the initial source information for the people chosen. This is published on wordpress.org/news. There is further collaboration on this in terms of translation, cross-linking, and adding locale links.

I think there a few others too, and good to see Jonathan's message above too.

**Comment by abhansnuk** [comment]
Hello, is there more detailed information of the project's vision or goals for 2023 which we could draw marketing items from or help to inspire ideas for the marketing team goals? Thanks.

**Comment by sereedmedia** [comment]
My primary 2023 goals for the marketing team: 

- develop meaningful ways for contributors to the marketing team to participate in the actual marketing and communications of the wordpress project 
- develop meaningful issues for first-time contributors that are not related to team management (ie note taking)
- bring additional elements of marketing (such as communications, messaging, UX, SEO) into the team purview
- clarify and standardize the processes for ongoing dot-org marketing activities
- identify project-wide opportunities for the marketing team to collaborate with other teams (particularly meta, doc, and training)
- strengthen project/cross-team communication and processes (internal marketing)
- align the activities of the community marketing team and the internal a8c dot-org marketing team
- setup a system where marketing can help update and revise dot-org content
